### DISCUSSION TITLE
v1.7.1

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#279] [Release]
**Changes and fixes:**
Compare UID from connect on activate ([#175](https://github.com/R2Northstar/NorthstarLauncher/pull/175))
Log UID in a variety of places during auth process ([#174](https://github.com/R2Northstar/NorthstarLauncher/pull/174))
Block `emit` command when `sv_cheats` is disabled ([#170](https://github.com/R2Northstar/NorthstarLauncher/pull/170))
Don't allow multiple of the same account to connect without a commandline arg ([#161](https://github.com/R2Northstar/NorthstarLauncher/pull/161))

**Backend:**
Reject non-string values in server/update_values ([#74](https://github.com/R2Northstar/NorthstarMasterServer/pull/74))
Move setting heartbeat so it can't be changed by the request data ([#68](https://github.com/R2Northstar/NorthstarMasterServer/pull/68))
Encode usernames so that they are valid URI strings ([#70](https://github.com/R2Northstar/NorthstarMasterServer/pull/70))
Guard against duplicate ports ([#67](https://github.com/R2Northstar/NorthstarMasterServer/pull/67))
Query game port on server create ([#63](https://github.com/R2Northstar/NorthstarMasterServer/pull/63))
Fix modInfo not iterable maybe ([#62](https://github.com/R2Northstar/NorthstarMasterServer/pull/62))
uid<->username lookup routes ([#36](https://github.com/R2Northstar/NorthstarMasterServer/pull/36))
Exposing player data (layouts and game stats) through endpoints. ([#48](https://github.com/R2Northstar/NorthstarMasterServer/pull/48))

**Contributors:**
@ASpoonPlaysGames @barnabwhy @BobTheBob9 @emma-miler @GeckoEidechse

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.7.1'>v1.7.1</a>.</em>

**Comment by Saltay2822** [comment]
what did i do wrong
![image_2022-05-20_180030472](https://user-images.githubusercontent.com/75194904/169622609-83d8775e-c981-4564-9227-21c795b67b4f.png)

**Reply by GeckoEidechse** [reply]
We just restarted master server due to some issues with authentication. It should work again now. If not open a ticket in our [Discord server](https://northstar.tf/discord)

**Reply by BT7376** [reply]
I'm having a similar error, when i start the game this code happens:
" [01:49:31] [error] CLIENT SCRIPT COMPILE ERROR Undefined variable "IsNorthstarServer"
  [01:49:31] [error] ui/openinvites.nut line [205] column [50] "

And so it leave's me with the animated html but not the menu options, images bellow show the code in game and the menus screen:
![20220522014952_1](https://user-images.githubusercontent.com/72712844/169679419-ab25a890-d72a-43db-8c3f-1dafe6fde6e3.jpg)
![20220522015041_1](https://user-images.githubusercontent.com/72712844/169679422-1234e985-c186-4a91-807b-9e9a178e672a.jpg)

**Reply by x3Karma** [reply]
Seems like you accidentally turned off one of the core mods.

Seek #help on Discord and we can help you easier.

**Reply by BT7376** [reply]
thank you, I'm going to seek help right away.

**Comment by thy-holy-potato** [comment]
how do install it into my steam tf2 I dont know how and the yt vids dont work for me

**Reply by GeckoEidechse** [reply]
I'd suggest using one of the [installers listed on the wiki](https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/northstar-installers).

If you still have problems, open a ticket in the `#help` channel on the [Northstar Discord server](https://northstar.tf/discord).
